spec/classroom_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../classroom'
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/217902519-2f7d6499-bbf8-4604-86c4-e6a3ce57e407.png)

- [ ] You didn't test **add_students** method shown in the image above :arrow_up: Kindly test this test to follow the requirements and fully test your classes :+1:

**Hint: You can see the below test to get an idea :arrow_down:**

```ruby
    it 'expect the classroom to have students' do
      student = Student.new(23, 'Maxwell', true)
      classroom.adding_students(student)
      expect(classroom.students).to eq [student]
    end
```